nengo_gui/config.py

```diff
@@ -43,10 +43,14 @@ def dumps(self, uids):
                                  % (uid, self[obj].expanded))
                     lines.append('_viz_config[%s].has_layout=%s'
                                  % (uid, self[obj].has_layout))
-            elif isinstance(obj, nengo_gui.components.component.Template):
+            elif isinstance(obj, nengo_gui.components.component.Component):
                 lines.append('%s = %s' % (uid, obj.code_python(uids)))
                 for k in obj.config_params.keys():
                     v = getattr(self[obj], k)
-                    lines.append('_viz_config[%s].%s = %g' % (uid, k, v))
+                    if isinstance(v, bool):
+                        val = '%s' % v
+                    else:
+                        val = '%g' % v
```

Comment:
Hmm...  or I could do

```python
val = ('%s' if isinstance(v, bool) else '%g') % v
```

but that might not be all that readable either....   I've generally tried to only use the inline if/else in situtations where there's only one thing going on....  this is a borderline case where I'm not only doing the if/else but also the string reformatting, so I think it's slightly clearer separated like this.